client/src/components/Search/Search.scss

```diff
@@ -58,4 +64,13 @@
     .page-link {
         border: 0 !important;
     }
+}
+
+.map-results {
+    .leaflet-container {
+        position: sticky !important;
```

Comment:
What does `!important` do?

Reply:
There's an explanation here
https://css-tricks.com/when-using-important-is-the-right-choice/

It is generally bad practice to use !important because it makes it very difficult to then override your styles from another place. However, I think it's ok here for two reasons:

1. The `.leaflet-container` is having its position set to `relative` with in-line styles, which gives it a high importance making it hard to override without using `!important`
2. This CSS will only be loaded on the Search component, so other components that use the `.leaflet-container` will not even be affected by this or have to attempt to override the `!important`